tasks/build-options/validateConfigs.js

```diff
@@ -68,7 +74,8 @@ async function validateFile (filePath) {
     for (const error of validate.errors) {
       invalidJsonFiles.push(error)
       console.error(`${prog}: ERROR: ${error.instancePath} ${error.message}`)
-      if (argv.mode === 'verbose') console.error(error)
+      // TOD: Add verbose mode with the full error:
```

Comment:
Is this change related to the GITC stuff?

Reply:
I have removed these with the latest commit